### DISCUSSION TITLE
Cp Editor not open

### DISCUSSION THREAD

**Discussion by md-shamim-ahmad** [cpeditor/cpeditor#1125] [Q&A] [answered]
I use cpeditor from 2020.
But it's not happen with me before install ubuntu 22.04 LTS.
After install ubuntu 22.04 LTS, cpeditor not open in my desktop.
I do not find any solution that why this editor not open in my desktop.

My desktop kernel version is 5.15.0-56-generic

**Comment by coder3101** [comment]
22.04 uses wayland as default and snap version does not runs on wayland. Please either compile from source or use xorg session to launch cpeditor. 

https://github.com/cpeditor/cpeditor/issues/1083#issuecomment-1156678967

**Reply by md-shamim-ahmad** [reply]
Thanks for your reply.
In previous, I also tried that way but it didn't work for me.

**Reply by coder3101** [reply]
Are you running from snap?

**Reply by md-shamim-ahmad** [reply]
Yes

**Comment by md-shamim-ahmad** [comment, marked as answer]
Hey thanks for your replay
Now it's work good.
Because I uninstall ```cpeditor``` and then download ```cpeditor.dev``` file and install it.
